### PR TITLE
net-misc/apifox: new package

### PR DESCRIPTION
**Pull request by gouwazi**

### CODE REVIEW

File: net-misc/apifox/apifox-2.2.19.ebuild [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit desktop
```
Comment (Universebenzene) [inline net-misc/apifox/apifox-2.2.19.ebuild]: 不inherit xdg的话merge完会不会报warning？

Comment (gouwazi) [reply net-misc/apifox/apifox-2.2.19.ebuild]: 不会

**Comment by gouwazi** [comment]
[ebuild   R   #] net-misc/apifox-2.2.19::gentoo-zh  0 KiB

Total: 1 package (1 reinstall), Size of downloads: 0 KiB

Would you like to merge these packages? [Yes/No] 

>>> Verifying ebuild manifests

>>> Emerging (1 of 1) net-misc/apifox-2.2.19::gentoo-zh
 * 2.2.19-amd64.zip BLAKE2B SHA512 size ;-) ...                                                                                                                                                                                      [ ok ]
>>> Unpacking source...
>>> Unpacking 2.2.19-amd64.zip to /var/tmp/portage/net-misc/apifox-2.2.19/work
>>> Source unpacked in /var/tmp/portage/net-misc/apifox-2.2.19/work
>>> Preparing source in /var/tmp/portage/net-misc/apifox-2.2.19/work ...
>>> Source prepared.
>>> Configuring source in /var/tmp/portage/net-misc/apifox-2.2.19/work ...
>>> Source configured.
>>> Compiling source in /var/tmp/portage/net-misc/apifox-2.2.19/work ...
>>> Source compiled.
>>> Test phase [not enabled]: net-misc/apifox-2.2.19

>>> Install net-misc/apifox-2.2.19 into /var/tmp/portage/net-misc/apifox-2.2.19/image
>>> Completed installing net-misc/apifox-2.2.19 into /var/tmp/portage/net-misc/apifox-2.2.19/image

 * Final size of build directory: 150532 KiB (147.0 MiB)
 * Final size of installed tree:  150544 KiB (147.0 MiB)

 * QA Notice: RESTRICT=binchecks prevented checks on these ELF files:
 * /opt/apifox/bin/Apifox.AppImage

>>> Installing (1 of 1) net-misc/apifox-2.2.19::gentoo-zh
 * checking 3 files for package collisions
>>> Merging net-misc/apifox-2.2.19 to /
--- /usr/
--- /usr/share/
--- /usr/share/pixmaps/
>>> /usr/share/pixmaps/apifox.svg
--- /usr/share/applications/
>>> /usr/share/applications/apifox.desktop
--- /opt/
--- /opt/apifox/
--- /opt/apifox/bin/
>>> /opt/apifox/bin/Apifox.AppImage
>>> Safely unmerging already-installed instance...
No package files given... Grabbing a set.
--- replaced obj /usr/share/pixmaps/apifox.svg
--- replaced dir /usr/share/pixmaps
--- replaced obj /usr/share/applications/apifox.desktop
--- replaced dir /usr/share/applications
--- replaced dir /usr/share
--- replaced dir /usr
--- replaced obj /opt/apifox/bin/Apifox.AppImage
--- replaced dir /opt/apifox/bin
--- replaced dir /opt/apifox
--- replaced dir /opt
>>> Original instance of package unmerged safely.
>>> net-misc/apifox-2.2.19 merged.

 * GNU info directory index is up-to-date.

**Comment by gouwazi** [comment]
```bash
[ebuild   R   #] net-misc/apifox-2.2.19::gentoo-zh  0 KiB

Total: 1 package (1 reinstall), Size of downloads: 0 KiB

Would you like to merge these packages? [Yes/No] 

>>> Verifying ebuild manifests

>>> Emerging (1 of 1) net-misc/apifox-2.2.19::gentoo-zh
 * 2.2.19-amd64.zip BLAKE2B SHA512 size ;-) ...                                                                                                                                                                                      [ ok ]
>>> Unpacking source...
>>> Unpacking 2.2.19-amd64.zip to /var/tmp/portage/net-misc/apifox-2.2.19/work
>>> Source unpacked in /var/tmp/portage/net-misc/apifox-2.2.19/work
>>> Preparing source in /var/tmp/portage/net-misc/apifox-2.2.19/work ...
>>> Source prepared.
>>> Configuring source in /var/tmp/portage/net-misc/apifox-2.2.19/work ...
>>> Source configured.
>>> Compiling source in /var/tmp/portage/net-misc/apifox-2.2.19/work ...
>>> Source compiled.
>>> Test phase [not enabled]: net-misc/apifox-2.2.19

>>> Install net-misc/apifox-2.2.19 into /var/tmp/portage/net-misc/apifox-2.2.19/image
>>> Completed installing net-misc/apifox-2.2.19 into /var/tmp/portage/net-misc/apifox-2.2.19/image

 * Final size of build directory: 150532 KiB (147.0 MiB)
 * Final size of installed tree:  150544 KiB (147.0 MiB)

 * QA Notice: RESTRICT=binchecks prevented checks on these ELF files:
 * /opt/apifox/bin/Apifox.AppImage

>>> Installing (1 of 1) net-misc/apifox-2.2.19::gentoo-zh
 * checking 3 files for package collisions
>>> Merging net-misc/apifox-2.2.19 to /
--- /usr/
--- /usr/share/
--- /usr/share/pixmaps/
>>> /usr/share/pixmaps/apifox.svg
--- /usr/share/applications/
>>> /usr/share/applications/apifox.desktop
--- /opt/
--- /opt/apifox/
--- /opt/apifox/bin/
>>> /opt/apifox/bin/Apifox.AppImage
>>> Safely unmerging already-installed instance...
No package files given... Grabbing a set.
--- replaced obj /usr/share/pixmaps/apifox.svg
--- replaced dir /usr/share/pixmaps
--- replaced obj /usr/share/applications/apifox.desktop
--- replaced dir /usr/share/applications
--- replaced dir /usr/share
--- replaced dir /usr
--- replaced obj /opt/apifox/bin/Apifox.AppImage
--- replaced dir /opt/apifox/bin
--- replaced dir /opt/apifox
--- replaced dir /opt
>>> Original instance of package unmerged safely.
>>> net-misc/apifox-2.2.19 merged.

 * GNU info directory index is up-to-date.
 ```

File: net-misc/apifox/apifox-2.2.19.ebuild [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+BDEPEND="app-arch/unzip"
+
+S="${WORKDIR}"
+
+src_install() {
+	into /opt/apifox
+	dobin "${S}/Apifox.AppImage"
+	domenu "${FILESDIR}/apifox.desktop"
+	doicon "${FILESDIR}/apifox.svg"
```
Comment (Universebenzene) [inline net-misc/apifox/apifox-2.2.19.ebuild]: svg矢量图标的话是不是用 `-s scalable` 会更好一点？

File: net-misc/apifox/apifox-2.2.19.ebuild (hunk 1 above)
Comment (gouwazi) [reply net-misc/apifox/apifox-2.2.19.ebuild]: 加上-s scalable就会警告我没有清除icon缓存，还是要加xdg

File: net-misc/apifox/apifox-2.2.19.ebuild (hunk 2 above)
Comment (Universebenzene) [reply net-misc/apifox/apifox-2.2.19.ebuild]: 咦，难道是用scalable就得inherit xdg了么，以前还真没注意过

**Pull request merged**